docs/conf.py

```diff
@@ -44,13 +45,12 @@
 # -- Options for HTML output -------------------------------------------------
 
 html_static_path = ["_static"]
-html_theme = "pydata_sphinx_theme"
+html_theme = "furo"
```

Comment:
That would be great if we would have the same theme accross all the packages related to onnx (onnx, onnxruntime, onnx-script, ...).

Reply:
Is furo a theme we like? If so I can replicate it in onnx. ORT so far is using the default theme when I checked. Or is there a desire to create our own theme like pytorch and tf did?